atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderRuntimeConfig.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.config;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonDeserialize(as = ImmutableLeaderRuntimeConfig.class)
+@JsonSerialize(as = ImmutableLeaderRuntimeConfig.class)
+@Value.Immutable
+public abstract class LeaderRuntimeConfig {
+
+    @Value.Default
+    public boolean onlyLogOnQuorumFailure() { return true; }
```

Comment:
I'm guessing we plan to make more things configurable here? Otherwise this seems a bit overkill

Reply:
Yes, I also want to open a separate pr to `PaxosLeadershipCreator` for live reloading paxos ping rates, which will be passed in LeaderRuntimeConfig.